modules/security-aai/pom.xml

```diff
@@ -13,6 +13,7 @@
   <properties>
     <opencast.basedir>${project.basedir}/../..</opencast.basedir>
     <checkstyle.skip>false</checkstyle.skip>
+    <java.release>8</java.release>
```

Comment:
What is the AAI code doing that restricts it to JDK 8?

Reply:
Test failures based on the Spring test library it is using.